src/components/TableBody.js

```diff
@@ -11,6 +14,21 @@ class TableBody extends Component {
     var lockstate = this.props.lockstate || 'unlocked';
     var total_working_hours = this.props.totaldays * 8;
     var total = 0;
+    // clear weekends from data
+    var weekday = [];
+    var i = 0;
+    Object.values(data).forEach(value => {
+      console.log('index is : ' + i)
```

Comment:
Städa bort den här?

Reply:
fixed in https://github.com/codelabsab/timereport-ui/pull/8/commits/968d0e637092d5286ac590562b5526da9fe0d1c9